website/docs/d/pi_volume_onboardings.html.markdown

```diff
@@ -40,10 +40,11 @@ Review the argument references that you can specify for your data source.
 ## Attribute reference
 In addition to all argument reference list, you can access the following attribute references after your data source is created. 
 
-- `onboardings` - (List of objects) - The list of volume onboardings.
+- `onboardings` - (List) List of volume onboardings.
 
   Nested scheme for `onboardings`:
       - `description` - (String) The description of the volume onboarding operation.
       - `id` - (String) The type of cycling mode used.
-      - `input_volumes` - (List of strings) List of volumes requested to be onboarded.
-      - `status` (String) Tthe status of volume onboarding operation.
\ No newline at end of file
+      - `input_volumes` - (List) List of volumes requested to be onboarded.
+      - `status` (String) The status of volume onboarding operation.
+      
```

Comment:
```suggestion
```suggestion
      - `input_volumes` - (List) List of volumes requested to be onboarded.
      - `status` (String) The status of volume onboarding operation.
```
```